src/main/java/lk/gov/health/phsp/bean/QueryComponentController.java

```diff
@@ -2701,7 +2702,7 @@ public Jpq createClientQuery(QueryComponent qc, Area ccArea, Date ccFrom, Date c
 
             jpql.setJselect("");
             jpql.setJfrom(ss);
-            jpql.setJwhere(w1 + w2 + w3 + " and c.retired=:f ");
+            jpql.setJwhere(w1 + w2 + w3 + " and c.retired=:f and c.reservedClient<>:res ");
```

Comment:
need to check parameter is provided

Reply:
This place as well. same above parameter. these occurrences in three if conditions